scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -2054,19 +2081,37 @@ then
 
         set -e
 
+        # Postpone script exit until all errors are reported
+        do_exit=0
         if [ ${RC[0]} -ne 0 ]; then
             wsrep_log_error "******************* FATAL ERROR ********************** "
+            # This is the only way, we can guess if it was the network connectivity problem.
+            # If socat gives up after -T seconds of stall, it exits with no error
+            # and pxb prints "OS errno 32 - Broken pipe" messages to its error log
+            # and then exits with code 1.
+            if cat "${DATA}/innobackup.backup.log" | grep -q "OS errno 32 - Broken pipe"; then
```

Comment:
Instead of `cat` followed by `grep`, can we not just use `grep` directly on the file?

Reply:
done